### ISSUE TITLE
We shouldn't show age dialog on /view page

### ISSUE THREAD

**Issue by poorvasingal** [code-dot-org/dance-party#291]
![image](https://user-images.githubusercontent.com/16227893/48377747-756d4600-e683-11e8-9799-5aabd40d389a.png)

This will be more of an issue when we make "view code" as a button on share page.

**Comment by ryansloan** [comment]
Yeah, I don't think we _must_ fix pre-HOC, but it would be nice. Since the dropdown is exposed as editable, we need to prompt you for your age.
Flagging as p2.
